src/tasks8/MusicianFirst.java

```diff
@@ -0,0 +1,21 @@
+package tasks8;
+
+public class MusicianFirst implements Piano {
+    public MusicianFirst(String name){
+        this.name = name;
+    }
+    @Override
+    public String sound(){
+        return getName() + " can play on " + "Piano, and piano sounds sweet, round, dark, and rich.";
+    }
```

Comment:
Тут потрібно викликати дефолтну реалізацію з  інтерфейсу
```suggestion
        return getName() + " can play on " + super.sound();
    }
```